src/components/IncentiveClaimItem/index.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { useIntl } from 'react-intl';
+import { CustomTooltip, TokenIcon, useThemeContext } from '@aave/aave-ui-kit';
+
+import Value from '../basic/Value';
+import Link from '../basic/Link';
+
+import defaultMessages from '../../defaultMessages';
+import staticStyles from './style';
+
+import tribeIcon from '../../images/tirbe.svg';
```

Comment:
should tribe be supported in ui-kit?, seems weird to custom handle it here

Reply:
@Argeare5